clang/test/dpct/query_api_mapping/cuDNN/dnn_part1.cu

```diff
@@ -0,0 +1,81 @@
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=cudnnCreate | FileCheck %s -check-prefix=cudnnCreate
+// cudnnCreate: CUDA API:
+// cudnnCreate-NEXT:   cudnnCreate(h /*cudnnHandle_t **/);
+// cudnnCreate-NEXT: Is migrated to:
+// cudnnCreate-NEXT:   h->create_engine();
+// cudnnCreate-EMPTY:
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=cudnnActivationBackward | FileCheck %s -check-prefix=cudnnActivationBackward
+// cudnnActivationBackward: CUDA API:
+// cudnnActivationBackward-NEXT: cudnnActivationBackward(
+// cudnnActivationBackward-NEXT:     h /*cudnnHandle_t*/, desc /*cudnnActivationDescriptor_t*/,
+// cudnnActivationBackward-NEXT:     alpha /*void **/, dst_d /*cudnnTensorDescriptor_t*/, dst /*void **/,
+// cudnnActivationBackward-NEXT:     diff_dst_d /*cudnnTensorDescriptor_t*/, diff_dst /*void **/,
+// cudnnActivationBackward-NEXT:     src_d /*cudnnTensorDescriptor_t*/, src /*void **/, beta /*void **/,
+// cudnnActivationBackward-NEXT:     diff_src_d /*cudnnTensorDescriptor_t*/, diff_src /*void **/);
+// cudnnActivationBackward-NEXT: Is migrated to:
+// cudnnActivationBackward-NEXT:     h.async_activation_backward(desc, *alpha, dst_d, dst, diff_dst_d, diff_dst, src_d, src, *beta, diff_src_d, diff_src);
+// cudnnActivationBackward-EMPTY:
+
+// RUN: dpct --cuda-include-path="%cuda-path/include" --query-api-mapping=cudnnActivationForward | FileCheck %s -check-prefix=cudnnActivationForward
+// cudnnActivationForward: CUDA API:
+// cudnnActivationForward-NEXT: cudnnActivationForward(
+// cudnnActivationForward-NEXT:     h /*cudnnHandle_t*/, desc /*cudnnActivationDescriptor_t*/,
+// cudnnActivationForward-NEXT:     alpha /*void **/, src_d /*cudnnTensorDescriptor_t*/, src /*void **/,
+// cudnnActivationForward-NEXT:     beta /*void **/, dst_d /*cudnnTensorDescriptor_t*/, dst /*void **/);
```

Comment:
here in migrated code,  h is not clear:  1.  type info in sycl side.  2. where is from, sycl  or dpct::. 

how about we add:  cudnnHandle_t  h in "//begin and //end",  then h will be more clear.